Test.C

```diff
@@ -11,108 +11,214 @@
 #define NUM_ROUNDS 5
 
 //Define structures
-struct Equipment {
-    char name[NAME_LENGTH]; // name of the piece of Equipment
+struct Equipment
+{
+    char name[NAME_LENGTH];               // name of the piece of Equipment
     char description[DESCRIPTION_LENGTH]; // short description of Equipment
-    int cost; // cost of the Equipment
-    int messageAmount; // number of messages able to be sent
+    int cost;                             // cost of the Equipment
+    int messageAmount;                    // number of messages able to be sent
 };
 
-struct Encryption {
-    char name[NAME_LENGTH]; // name of the encryption method
+struct Encryption
+{
+    char name[NAME_LENGTH];               // name of the encryption method
     char description[DESCRIPTION_LENGTH]; // short description of how encryption works
-    int cost; // cost of the encryption
-    double effectiveness; // how effective the encryption is (percentage)
-
+    int cost;                             // cost of the encryption
+    double effectiveness;                 // how effective the encryption is (percentage)
 };
 
-struct Attack {
-    char name[NAME_LENGTH]; // name of the attack
+struct Attack
+{
+    char name[NAME_LENGTH];               // name of the attack
     char description[DESCRIPTION_LENGTH]; // short description of how the attack works
-    int tier; // level of intensity
+    int tier;                             // level of intensity
 };
 
-struct EncryptionInventory {
-    Encryption encryption; // the data
+struct EncryptionInventory
+{
+    Encryption encryption;     // the data
     EncryptionInventory *next; // the pointer
 };
 
-struct Player {
-    char name[NAME_LENGTH]; // the player's name
+struct Player
+{
+    char name[NAME_LENGTH];                      // the player's name
     int equipmentInventory[NUM_EQUIPMENT] = {0}; // the current equipment the user owns
-    EncryptionInventory encryptionInventory; // the current encryption methods the user owns
-    int currentCredits; // the current number of credits the user has available
+    EncryptionInventory encryptionInventory;     // the current encryption methods the user owns
+    int currentCredits;                          // the current number of credits the user has available
 };
 
-struct Round {
+struct Round
+{
     int attackTier;
     Attack attack;
 };
 
 //Function prototypes
 void displayBriefingMessage(Player *playerPtr);
 
-int main(){
+void equipmentStore()
```

Comment:
Functions should not be declared before main for readability sake. Use a function prototype before main. You will need to pass in the player structure as we want to avoid globals. This will have to be a pointer I believe.
```suggestion
void equipmentStore(struct Player *playerPtr)
```